packages/stog/stog.0.20.0/opam

```diff
@@ -0,0 +1,54 @@
+opam-version: "2.0"
+synopsis:
+  "Static web site compiler, able to handle blog posts as well as regular pages or any XML document in general"
+maintainer: "zoggy@bat8.org"
+authors: "Zoggy <zoggy@bat8.org>"
+license: "GPL-3.0-only"
+homepage: "https://www.good-eris.net/stog/"
+doc: "https://www.good-eris.net/stog/doc.html"
+bug-reports: "https://framagit.org/zoggy/stog/issues"
+depends: [
+  "dune" {>= "2.9"}
+  "ocaml" {>= "4.12.0" & < "4.13"}
```

Comment:
Just curious, why is it not compatible with 4.13?

Reply:
The CI logs has been removed now that the package is ok, but IIRC this is due to a change in OCaml's internals, called from the stog part embedding the toplevel.

Reply:
Ah, you’re using `Toploop`. I see, thanks.